pages/task/read/views/project.jsx

```diff
@@ -117,7 +118,8 @@ const Project = ({ task, project, establishment, children, schema, formFields })
                     declarationConfirmed(declarations.awerb) &&
                       <Fragment>
                         <p>
-                          <Snippet>declarations.awerb.review-date</Snippet> {declarations['awerb-review-date']}
+                          <Snippet>declarations.awerb.review-date</Snippet>
+                          <ReactMarkdown>{declarations['awerb-review-date']}</ReactMarkdown>
```

Comment:
@wheelsandcogs I think Snippet renders as markdown by default - you probably only need one snippet and pass the review data in as a variable

Reply:
I think that's a label and then a value, so this seems correct to me.

However, rendering `Markdown` inside a `<p>` is likely to result in some mild weirdness because the markdown will render its own `<p>` tags (which is the point).

Reply:
sorted.